README.md

```diff
@@ -24,30 +24,43 @@ Note: If you do not specify `use_codedeploy = true`, the above process will not
 Also Note: CodePipeline and CodeDeploy cannot be used together to deploy a Lambda. If you are using CodePipeline, you cannot specify `use_codedeploy = true`. CodeDeploy works fine with other pipelining tools (e.g. GitHub Actions).
 
 ## Usage
+For a Zip file lambda
 ```hcl
 module "lambda_api" {
-  source                        = "github.com/byu-oit/terraform-aws-lambda-api?ref=v1.3.0"
-  app_name                      = "my-lambda"
-  env                           = "dev"
-  codedeploy_service_role_arn   = module.acs.power_builder_role.arn
-  lambda_zip_file               = "./src/lambda.zip"
-  handler                       = "index.handler"
-  runtime                       = "nodejs12.x"
+  source       = "github.com/byu-oit/terraform-aws-lambda-api?ref=v2.0.0"
```

Comment:
My only comment here would be to maybe format this so if someone copies it, it will pass the `terraform fmt` step.

Reply:
it's formatted correctly, I think the diff here looks odd in GH for some reason.